transport.go

```diff
@@ -124,19 +124,17 @@ func NewTransport(key ic.PrivKey, psk pnet.PSK, gater connmgr.ConnectionGater) (
 	if _, err := io.ReadFull(keyReader, config.StatelessResetKey); err != nil {
 		return nil, err
 	}
+	config.Tracer = tracer
```

Comment:
Is it ok for this to be nil? i.e. when the `QLOGDIR` env var is not set?

Reply:
Yes, that's ok. It's also ok if the callback returns a nil `io.WriteCloser` (that way you could sample only a fraction of the connections you're handling, for example).